docs/release.rst

```diff
@@ -23,15 +23,18 @@ Enhancements
   SQLite database to be used as the backing store for an array or group.
   By :user:`John Kirkham <jakirkham>`, :issue:`368`, :issue:`365`.
 
+* Efficient iteration over arrays by decompressing chunkwise.
+  By :user:`Jerome Kelleher <jeromekelleher>`, :issue:`398`.
```

Comment:
```suggestion
  By :user:`Jerome Kelleher <jeromekelleher>`, :issue:`398`, :issue:`399`.
```

Reply:
Tiny thing, I'll resolve later.

Reply:
Cool --- wasn't sure if this was referring to the issue or the PR  as well.